README.rst

```diff
@@ -193,11 +193,11 @@ impact to other applications will be minimal.
 Upgrading an existing Synapse
 =============================
 
-The instructions for upgrading synapse are in `UPGRADE.rst`_.
+The instructions for upgrading synapse are in `the upgrade notes`_.
 Please check these instructions as upgrading may require extra steps for some
 versions of synapse.
 
-.. _UPGRADE.rst: UPGRADE.rst
+.. _the upgrade notes: https://matrix-org.github.io/synapse/develop/upgrading
```

Comment:
`upgrading/README.md` ?

Reply:
mdbook will transform `README.md` files to `index.html`, and github will resolve `/synapse/develop/upgrading` to `/synapse/develop/upgrading/index.html` for us.